cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt

```diff
@@ -101,8 +101,8 @@ internal interface CampaignManager {
     val hasLocalData: Boolean
 
     // dateCreated
-    var gdprDateCreated: Instant?
-    var ccpaDateCreated: Instant?
+    var gdprDateCreated: String?
+    var ccpaDateCreated: String?
```

Comment:
@carmelo-iriti was the `Instant` type causing the issue?

Reply:
yes